packages/observable/src/index.ts

```diff
@@ -0,0 +1,95 @@
+import { Store, Atom, Action } from '@reatom/core'
+
+export type ActionOrValue<T> = T extends undefined ? Action<any, string> : T
+
+export interface Observable<T> {
+  /**
+   * Subscribes observer for Store or Atom<T> updates
+   *
+   * @return {Subscription}
```

Comment:
Maybe is better use `@returns`?

Reply:
Why? I think it's not matter

Reply:
In jsdocs `@return` and `@returns` is equivalent code

But it's about meaning when reading code.

Example with translete to russian:
```
return - вернуть
returns - возвращается
```

Reply:
This is not critical part.You don't have to edit it.